cmd/createBuildPipelineJob.go

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2023
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cmd
+
+import (
+	"errors"
+
+	"github.com/equinor/radix-cli/generated-client/client/application"
+	"github.com/equinor/radix-cli/generated-client/models"
+	"github.com/equinor/radix-cli/pkg/client"
+	"github.com/equinor/radix-cli/pkg/flagnames"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// createBuildPipelineJobCmd represents the createBuildPipelineJob command
+var createBuildPipelineJobCmd = &cobra.Command{
+	Use:   "build",
+	Short: "Will trigger build of a Radix application",
+	Long:  `Triggers build of Radix application, for branches that are mapped to a environment in the Radix config`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		appName, err := getAppNameFromConfigOrFromParameter(cmd, flagnames.Application)
+		if err != nil {
+			return err
+		}
+		if appName == nil || *appName == "" {
+			return errors.New("application name is required")
+		}
```

Comment:
Or add following 
```
	if err := createBuildPipelineJobCmd.MarkFlagRequired(flagnames.Application); err != nil {
		log.Fatalf("Error during command initialization: %v", err)
	}
```
instead of 
```
   if appName == nil || *appName == "" {
			return errors.New("application name is required")
		}
```
to be consistent